app/src/organisms/OnDeviceDisplay/Navigation/NavigationMenu.tsx

```diff
@@ -29,39 +36,48 @@ export function NavigationMenu(props: NavigationMenuProps): JSX.Element {
         key="home-gantry"
```

Comment:
to align with the above text

```suggestion
        key="home-robot-arm"
```